lib/auth/mfa/session.go

```diff
@@ -34,5 +34,14 @@ type SSOMFASessionData struct {
 	// ConnectorType is SSO type of the corresponding Auth connector (SAML, OIDC).
 	ConnectorType string `json:"connector_type,omitempty"`
 	// ChallengeExtensions are Teleport extensions that apply to this SSO MFA session.
-	ChallengeExtensions *mfav1.ChallengeExtensions `json:"challenge_extensions,omitempty"`
+	ChallengeExtensions *ChallengeExtensions `json:"challenge_extensions"`
+}
+
+// ChallengeExtensions is a json struct for [mfav1.ChallengeExtensions].
+//
+// The UserVerificationRequirement field from [mfav1.ChallengeExtensions]
+// has been omitted as it's only relevant to WebAuthn/Passwordless.
+type ChallengeExtensions struct {
```

Comment:
Is this duplicated just so it omits UserVerificationRequirement? Instead of duping could we use wantypes.ChallengeExtensions and document that UserVerificationRequirement is not used for SSOMFASessionData? WDYT?

If not, then call this one SSOChallengeExtensions?

Reply:
Friendly ping for this one? Follow-up also?

Reply:
Ah right, I was able to move this to `lib/auth/mfa` and use it for both session data types.